File: src/app/lib/pages/lan/lan.dart
import 'package:flutter/widgets.dart';
import 'package:mirrorx/pages/page.dart';

class LanPage extends AppPage {
  const LanPage({Key? key, required String tag}) : super(key: key, tag: tag);

  @override
  _LanPageState createState() => _LanPageState();
}

class _LanPageState extends State<LanPage> {
  @override
  Widget build(BuildContext context) {
    return Container(
      child: Text("Lan Discovery"),
    );
  }
}

File: app/desktop/mirrorx/src/frame/page/lan.rs
use super::Page;
use crate::frame::state::UIState;

pub struct LanPage {}

impl LanPage {
    pub fn new() -> Self {
        Self {}
    }
}

impl Page for LanPage {
    fn draw(&mut self, ui: &mut eframe::egui::Ui, ui_state: &mut UIState) {
        ui.label("lan");
    }
}
